cmd/swarm/upload.go

```diff
@@ -55,12 +73,17 @@ func upload(ctx *cli.Context) {
 		defaultPath     = ctx.GlobalString(SwarmUploadDefaultPath.Name)
 		fromStdin       = ctx.GlobalBool(SwarmUpFromStdinFlag.Name)
 		mimeType        = ctx.GlobalString(SwarmUploadMimeType.Name)
+		debug           = ctx.GlobalBool("debug")
```

Comment:
This is a bit hidden and doesn't appear in the `--help` section from `swarm up`.